docs/config/processors/sitemap.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = function createSitemap() {
+  return {
+    $runAfter: ['paths-computed'],
+    $runBefore: ['rendering-docs'],
+    $process: function(docs) {
+      docs.push({
+        id: 'sitemap.xml',
+        path: 'sitemap.xml',
+        outputPath: '../sitemap.xml',
+        template: 'sitemap.template.xml',
+        urls: docs.filter(function(doc) {
+          return doc.path &&
+            doc.outputPath &&
+            doc.outputPath.indexOf('index') !== 0 &&
+            doc.outputPath.indexOf('examples/') !== 0 &&
+            doc.outputPath.indexOf('ptore2e/') !== 0;
```

Comment:
You could do this with a regex:

```
!doc.outputPath.test(/^index|examples\/|ptore2e\//)
```

(@gkalpak would be proud of me :-D )

Reply:
lol